README.md

```diff
@@ -0,0 +1,22 @@
+# demes-r
+R library for parsing Demes demographic models
+
+A demographic model describes one or more demes (or populations),
+how they change over time, and their relationships to one another.
+`demes` provides a self-contained and unambiguous in-memory description
+of a demographic model.
+
+Please see the [documentation](https://popsim-consortium.github.io/demes-docs/latest/index.html)
+for more details.
```

Comment:
Maybe omit this link for now? It might just be confusing for people thinking that this points to docs for the R library.

Reply:
That's a good point, I might add that it is the demes documentation to avoid that confusion. Generally, I would like to keep the link though so people can find more information about demes

Reply:
Ok, that sounds good.